crosswalks/BibTeX.csv

```diff
@@ -0,0 +1,74 @@
+Property,BibTeX (@softwareversion)
+codeRepository,repository
+programmingLanguage,
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,file
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version (1)
+storageRequirements,
+supportingData,
+author,author
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,date (2) (year) / (month)
+dateModified,date (3) (year) / (month)
+datePublished,date (1) (year) / (month)
+editor,editor
+encoding,
+fileFormat,
+funder,
+keywords,
+license,license
+producer,
+provider,
+publisher,publisher
+sponsor,
+version,version (2)
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,abstract
+identifier,doi / hal_id
+name,title
+sameAs,
+url,url
+relatedLink,
+givenName,author
+familyName,author
+email,
+affiliation,organization
+identifier,
+name,author

```

Comment:
given we also map `author,author`, this implies that "givenName of author" in Codemeta matches "author of author" in BibTeX, which does not really make sense. I believe we should remove these three name mappings, and leave crosswalks users to figure it out.

Reply:
I am not sure to understand what you mean here, would you have any example?
Do you mean that we should not map `author` at all?

Reply:
I mean that `author` should be mapped, but not `givenName`, `familyName`, or `name`